app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,29 @@
+class AnswersController < ApplicationController
+  before_action :set_comment
+  before_action :auth_user?
+
+  def create
+    @answer = @comment.answers.new(answer_params)
+    @answer.user = current_user
+    @answer.save
+    redirect_back(fallback_location: root_path)
+  end
+
+  def destroy
+    @answer = @comment.answers.find(params[:id])
+    @answer.destroy
+    redirect_back(fallback_location: root_path)
+  end
+
+  private
+
+  def answer_params
+    params.require(:answer).permit(:body, :comment_id)
+  end
+  
+
+  def set_comment
+    @comment = Comment.find(params[:comment_id])
```

Comment:
Add a guard if the comment is null

Reply:
For security, I already put this line of code on each model comment and answer : "validates :body, presence: true, allow_blank: false" so the comment will not be created if it is empty.
For the null, an answer depends on a comment and a comment depends on an event, so if the event is null, a comment can't exist, no way to create one. and with the event, in the create method, there is already a condition.

Reply:
got it, great!